src/js/urlencoded.js

```diff
@@ -0,0 +1,21 @@
+// Module to help express understand URL-encoded data.
+"use strict";
+var fluid  = fluid || require("infusion");
+var gpii   = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.express.middleware.bodyparser.urlencoded");
+
+var bp         = require("body-parser");
+
+gpii.express.middleware.bodyparser.urlencoded.getUrlencodedMiddlewareFunction = function() {
```

Comment:
This could just be named gpii.express.middleware.bodyparser.urlencoded.getMiddleware - since its purpose is clear from all the existing namespace elements. And a name of "getMiddleware" could become a standard convention for all of these functions